java/daikon/inv/unary/OneOf.java.jpp

```diff
@@ -2118,8 +2118,9 @@ public String format_csharp_contract(@GuardSatisfied ONEOFSCALAR this) {
     if (num_elts == 0) {
       return false;
     }
-
-    if (!(state instanceof INT[])) {
+    @SuppressWarnings("allcheckers:operand.instanceof.subtype") //  Warnings are suppressed because only the data type of the state needs to be checked and not the annotations
+    boolean stateTypeMatch = !(state instanceof INT[]);
```

Comment:
"stateTypeMatch" is a bad variable name because this is looking for a **mis**match.
I think "stateIsNotArray" would be a clearer variable name.